Shared/ViewModels/AdminDashboard/DevicesViewModel.swift

```diff
@@ -12,136 +12,78 @@ import JellyfinAPI
 import OrderedCollections
 import SwiftUI
 
-final class DevicesViewModel: ViewModel, Eventful, Stateful {
+@MainActor
+@Stateful
+final class DevicesViewModel: ViewModel {
 
-    // MARK: Event
-
-    enum Event {
-        case error(JellyfinAPIError)
-        case success
-    }
-
-    // MARK: - Action
-
-    enum Action: Equatable {
+    @CasePathable
+    enum Action {
         case refresh
         case delete(ids: [String])
```

Comment:
Have this action take a `Set<String>` instead so we don't need to cast to an `Array` on the view side.